ui/StatusQ/src/StatusQ/Layout/StatusAppNavBar.qml

```diff
@@ -1,195 +1,129 @@
+    property alias chatItemsModel: chatItemsListView.model
+    property alias chatItemDelegate: chatItemsListView.delegate
 
-    property var sectionModel: []
-    property string communityTypeRole: ""
-    property int communityTypeValue: -1
-    property int navBarButtonSpacing: 12
+    property alias communityItemsModel: communityItemsListView.model
+    property alias communityItemDelegate: communityItemsListView.delegate
 
-    property StatusNavBarTabButton navBarCameraButton 
-    property StatusNavBarTabButton navBarProfileButton
-    property Component regularNavBarButton
-    property Component communityNavBarButton
+    property alias regularItemsModel: regularItemsListView.model
+    property alias regularItemDelegate: regularItemsListView.delegate
 
-    property var filterRegularItem: function(item) { return true; }
-    property var filterCommunityItem: function(item) { return true; }
+    property real delegateHeight
 
-    signal aboutToUpdateFilteredRegularModel()
-    signal aboutToUpdateFilteredCommunityModel()
+    property alias cameraComponent: cameraItemLoader.sourceComponent
+    property alias profileComponent: profileItemLoader.sourceComponent
 
-    onNavBarProfileButtonChanged: {
-        if (!!navBarProfileButton) {
-            navBarProfileButton.parent = navBarProfileButtonSlot
-        }
+    implicitWidth: 78
+    implicitHeight: layout.implicitHeight
+
+    Rectangle {
+        anchors.fill: parent
+        color: Theme.palette.statusAppNavBar.backgroundColor
     }
 
-    onNavBarCameraButtonChanged: {
-        if (!!navBarCameraButton) {
-            navBarCameraButton.parent = navBarCameraButtonSlot
+    ColumnLayout {
+        id: layout
+
+        anchors {
+            fill: parent
+            topMargin: 48
+            bottomMargin: 24
         }
-    }
 
-    function triggerUpdate(){
-        navBarModel.update()
-    }
+        spacing: d.spacing
 
-    StatusAppNavBarFilterModel {
-        id: navBarModel
+        ListView {
+            id: chatItemsListView
 
-        filterAcceptsItem: filterRegularItem
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            Layout.minimumHeight: root.delegateHeight
+            Layout.preferredHeight: d.implicitListViewHeight(this)
+            Layout.maximumHeight: Layout.preferredHeight
 
-        model: statusAppNavBar.sectionModel
+            clip: true
+            spacing: d.spacing
+            boundsBehavior: contentHeight > height ? Flickable.DragAndOvershootBounds : Flickable.StopAtBounds
+        }
 
-        onAboutToUpdateFilteredModel: {
-            statusAppNavBar.aboutToUpdateFilteredRegularModel()
+        Rectangle {
+            implicitHeight: 1
+            Layout.preferredWidth: d.separatorWidth
+            Layout.alignment: Qt.AlignHCenter
+            color: Theme.palette.directColor7
+
+            visible: chatItemsListView.count && mmunityItemsListView.contentHeight > communityItemsListView.height
```

Comment:
Massive typo or broken line? `mmunityItemsListView`

Reply:
Good catch! Massive type :eyes: Have no clue how it worked :thinking: